### PR TITLE
[workgroups] Create testing workgroup

### PR DESCRIPTION
**Pull request by Mengerian**
This workgroup will provide a place to coordinate Upgrade testnet, and share information and parameters for people to connect to the upgrade testnet.

### CODE REVIEW

File: workgroups/wg-testing/2019-05-15_upgrade_testnet.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+user: (testnet address) password:(anything)
+
+Electrumx server: `testnet.imaginary.cash:50002` will follow the bitcoind.
+
+Electron Cash Schnorr edition: https://github.com/markblundeberg/Electron-Cash/tree/schnorr
+(only produces schnorr sigs)
+
+## Communication
+
+To communicate and coordinate with other Upgrade Testnet users, you can join the [Discord Group](https://discord.gg/sjncsXb).
```
Comment (deadalnix) [inline workgroups/wg-testing/2019-05-15_upgrade_testnet.md]: Seriously ?

Comment (Mengerian) [reply workgroups/wg-testing/2019-05-15_upgrade_testnet.md]: Do you prefer sticking with Telegram?

**Comment by bluffed84** [comment]
Commit

**Comment by bluffed84** [comment]
Committed

**Pull request merged**